shared/settings/feedback-container.js

```diff
@@ -47,27 +48,33 @@ class FeedbackContainer extends Component<void, {}, State> {
     if (isIOS) {
       // We don't get the notification from the daemon so we have to do this ourselves
       const logs = []
+      console.log('Starting log dump')
```

Comment:
some small feedback so we can see whats going on to further debug this if we need it